dotcom-rendering/src/components/AdBlockAsk.tsx

```diff
@@ -0,0 +1,167 @@
+import type { SerializedStyles } from '@emotion/react';
+import { css } from '@emotion/react';
+import { headline, textSans } from '@guardian/source-foundations';
+import { palette } from '@guardian/source-foundations/cjs/source-foundations/src/colour/palette';
+import ArrowRightIcon from '../static/icons/arrow-right.svg';
+import HandsLogoSmall from '../static/logos/hands-small.svg';
+import HandsLogo from '../static/logos/hands.svg';
+
+type AdBlockAskSize = 'leaderboard' | 'mpu';
+
+const linkStyles = css`
+	background: ${palette.brandAlt[400]};
+	border-radius: 16px;
+	box-sizing: border-box;
+	color: ${palette.neutral[7]};
+	${textSans.medium()};
+	font-weight: 700;
+	text-decoration: none;
+
+	float: left;
+	line-height: 18px;
+	position: relative;
+	height: 32px;
+	padding: 6px 34px 0 12px;
+
+	svg {
+		fill: currentColor;
+		position: absolute;
+		right: 3px;
+		top: 50%;
+		height: 32px;
+		width: 32px;
+		transform: translate(0, -50%);
+		transition: transform 0.3s ease-in-out;
+	}
+
+	:hover svg {
+		transform: translate(3px, -50%);
+	}
+`;
+
+const adBlockAskH2Styles = css`
+	${headline.xsmall()}
+	color: ${palette.brandAlt[400]};
+	font-weight: bold;
+`;
+
+const adBlockAskTextStyles: Record<AdBlockAskSize, SerializedStyles> = {
+	leaderboard: css`
+		margin-top: 8px;
```

Comment:
V.v.minor but margin and padding can be specified using the `space` utility from source:

https://theguardian.design/2a1e5182b/p/912e85-spacing/b/084803

```suggestion
		margin-top: ${space[2]}px;;
```

Reply:
Good shout. Have updated to use `space` where it felt appropriate.